### PR TITLE
Reconcile both event listeners to fix error messages not resurfacing

### PR DESCRIPTION
**Pull request by djgrant**

### CODE REVIEW

File: src/addListeners.js [hunk 1]
```diff
@@ -5,7 +5,11 @@ var validate = require('./validate');
 var getErrorElement = require('./helpers/getErrorElement');
 
 // Handle elements with data-validations properties
-$(document).on('keyup click change blur', 'input, textarea, select', function (e) {
+$(document).on('keyup click change blur', 'input, textarea, select', onChange);
+$(document).on('blur keyup', '[data-validations]', onChange);
```
Comment (mrcthms) [inline src/addListeners.js:9]: Can this not just be `$(document).on('keyup click change blur', 'input, textarea, select, [data-validations]', onChange);`

File: src/addListeners.js [hunk 2]
```diff
@@ -74,7 +52,16 @@ $(document).on('blur keyup', '[data-validations]', function (e) {
...
-});
+
+  // Validate and show errors
+  if (!validate.element(this)) {
+    var $error = getErrorElement($this);
+    $error.trigger('errorRemoved');
+  }
+
+  // Do not use getParentForm()
+  validate.form($(e.target).parents('form').get(0));
```
Comment (mrcthms) [inline src/addListeners.js:63]: can use `$input` here, rather than `$(e.target)`

File: src/addListeners.js [hunk 3]
```diff
@@ -5,7 +5,11 @@ var validate = require('./validate');
 var getErrorElement = require('./helpers/getErrorElement');
 
 // Handle elements with data-validations properties
-$(document).on('keyup click change blur', 'input, textarea, select', function (e) {
+$(document).on('keyup click change blur', 'input, textarea, select', onChange);
+$(document).on('blur keyup', '[data-validations]', onChange);
+
+function onChange(e) {
+  var $input = $(e.target);
```
Comment (callumacrae) [inline src/addListeners.js:12]: Isn't this the same as `$this`?

Comment (djgrant) [inline src/addListeners.js:12]: Yes, I reconciled the 2 event handlers but tried not to change the code until I have time to properly look through it.

**Pull request merged**